types/googlepay/googlepay-tests.ts

```diff
@@ -101,6 +101,14 @@ function addGooglePayButton() {
     // $ExpectError
     buttonOptions.buttonRootNode = document.createElement('div');
 
+    buttonOptions.buttonLocale = '';
+    buttonOptions.buttonLocale = undefined;
+    // $ExpectError
+    buttonOptions.buttonLocale = {};
+    buttonOptions.buttonLocale = 'en';
+    buttonOptions.buttonLocale = 'qw';
+    buttonOptions.buttonLocale = 'zh';
```

Comment:
Why would this return an error?

Reply:
`// $ExpectError` only applies to the next line, not the lines following it.